src/NUnitEngine/nunit.engine/Internal/AssemblyHelper.cs

```diff
@@ -21,6 +21,7 @@
 // WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 // ***********************************************************************
 
+#if !NETSTANDARD1_3
 using System;
 using System.IO;
 using System.Reflection;
```

Comment:
There's a TODO in this file in terms of making `GetAssemblyPathFromCodeBase` xplat. Probably can be a future issue.